packages/11ty/.eleventy.js

```diff
@@ -151,11 +116,16 @@ module.exports = function(eleventyConfig) {
   eleventyConfig.addPlugin(EleventyHtmlBasePlugin)
 
   /**
-   * Plugins are loaded in order of the `addPlugin` statements,
-   * plugins that mutate globalData must be added before other plugins
+   * Plugins are loaded in this order: 
+   *   1) immediately invoked
+   *   2) addPlugin statements
+   * Plugins that mutate globalData must be added before other plugins
```

Comment:
Expanding on this comment with more detail about how each of these plugins rely on data from the previous may be kind to our future selves.

Reply:
Yeah, it is confusing. I've updated the comment